share/spack/gitlab/cloud_pipelines/configs/win64/ci.yaml

```diff
@@ -0,0 +1,20 @@
+ci:
+  pipeline-gen:
+  - build-job:
+      after_script::
+      - Write-Output "Done"
+
+      before_script::
+      - fsutil 8dot3name set C:\ 0
+      - . .\share\spack\setup-env.ps1
+      - spack.ps1 compiler find
```

Comment:
This should not be needed in the build-job, the generate job does this and caches the result in the `spack.yaml` we activate here for the environment

Reply:
Addressed in latest push.